pages/linux/chattr.md

```diff
@@ -0,0 +1,15 @@
+# chattr
+
+> Change attributes of files or folders.
+
+- Make a file or folder immutable to changes and deletion, even by superuser:
+
+`chattr +i {{path}}`
+
+- Undo previous changes to make a file or folder mutable:
```

Comment:
Shouldn't this be just `make a file or folder mutable`? It doesn't really have anything to do with the previous change (file could have been created as immutable).